_tests_/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+echo "> Cloning FHIR Generated Patients..."
+git clone --single-branch --branch master https://github.com/projecttacoma/fhir-patient-generator.git
```

Comment:
The connectathon repo `https://github.com/DBCG/connectathon.git` also needs to be cloned.